cpp/src/parquet/level_conversion_inc.h

```diff
@@ -30,8 +30,7 @@
 #include "parquet/exception.h"
 #include "parquet/level_comparison.h"
 
-namespace parquet {
-namespace internal {
+namespace parquet::internal {
 #ifndef PARQUET_IMPL_NAMESPACE
 #error "PARQUET_IMPL_NAMESPACE must be defined"
 #endif
```

Comment:
We may want to use `namespace parquet::internal::PARQUET_IMPL_NAMESPACE`.

Reply:
Done